src/Service/OpenPlatform/SearchService.php

```diff
@@ -260,6 +260,9 @@ private function recursiveSearch(string $token, string $identifier, string $type
                         $extraISBN = $tools->convertIsbn13to10($identifier);
                     } elseif ($tools->isValidIsbn10($identifier)) {
                         $extraISBN = $tools->convertIsbn10to13($identifier);
+                    } else {
+                        // The ISBN was not found to be valid by the tool, so we can't convert it.
+                        $extraISBN = false;
```

Comment:
Would prefer that we just do `$extraISBN = false;` before the try-catch, so that `false` is the default state.

Reply:
Fixed